components/common/ScoringGuide.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheck, faQuestion } from '@fortawesome/free-solid-svg-icons'
+
+import Modal from '../modal/Modal'
+import ScoreLabel from './ScoreLabel'
+
+// TODO: Determine if there's a suitable free icon substitute for this?
+const EmptySquare = () => (
+    <span className='empty-square-icon' style={{fontSize: '2em', paddingBottom: '2px'}}>
+        &#9633;
+    </span>
+)
+
+const ScoringGuide = () => (
+    <Modal target="scoring-guide"
+        text="Scoring guide"
+        title="Scoring guide">
+        <div className="container-fluid">
+            <div className="row">
+                <h4 className="guide-heading">Overall State Score</h4>
+                <div className='overall scoring-guide'>
+                    {[4, 3, 2, 1, 0].map(score => (
+                        <ScoreDescription key={score} score={score}/>
+                    ))}
+                </div>
```

Comment:
For some reason the positioning of the overall scores are messed up on Chrome on my machine. Is this just me?

<img width="600" alt="Screen Shot 2021-02-21 at 6 18 02 PM" src="https://user-images.githubusercontent.com/9601737/108642089-325cc880-7471-11eb-9599-db4efd788c40.png">

Reply:
Yikes! I wonder what happened there. I'll take a look.

Reply:
Bad css selector. Will fix.